FreeRTOS/FreeRTOS
# Update for configIDLE_TASK_NAME longer than configMAX_TASK_NAME_LEN

Description
-----------
* Update unit test to cover the case that configIDLE_TASK_NAME is longer than configMAX_TASK_NAME_LEN.
* Update community supported demos submodule pointer for passive idle hook.

Test Steps
-----------


Checklist:
----------



Related Issue
-----------
Kernel PR [#784](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/784)


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
